src/main/ApplicationImpl.cpp

```diff
@@ -640,25 +644,23 @@ ApplicationImpl::getWorkerIOService()
     return mWorkerIOService;
 }
 
-std::vector<std::unique_ptr<Invariant>>
-ApplicationImpl::enabledInvariants() const
+void
+ApplicationImpl::enableInvariantsFromConfig()
```

Comment:
maybe use this PR to also ditch this way of configuring invariants?
I think it makes more sense to have a list of strings in config, that way we can extend to patterns in the future. It also allows to ditch the `kName` static variables (the name is only going to be used in `getName` at that point).